decorator.rb

```diff
@@ -0,0 +1,9 @@
+class Decorator
+  def initialize(nameable)
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
```

Comment:
- [ ] Kindly make sure that `Decorator` class inherits from `Namable` 📌 as per [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_decorator_v2.md)

_After implementing the changes, please make sure that a class is decorated as expected_